CONN-5815 | CONN-5842 - Claude Compliance C2C Source and App docs (#6713)
SumoLogic/sumologic-documentation: docs/integrations/saas-cloud/claude-compliance.md

---
id: claude-compliance
title: Claude Compliance
sidebar_label: Claude Compliance

> description: The Claude Compliance Monitoring app analyzes compliance, security, authentication, API, billing, integration, and Claude activity data from the Claude platform. It provides visibility into user behavior, operational activity, governance changes, and potential security threats across the environment.

Reviewer: Lead with Sumo Logic and lower character count for SEO

```suggestion
description: The Sumo Logic app for Claude Compliance analyzes activity, security, and compliance data from the Claude platform to support governance and threat detection.
```

---

import useBaseUrl from '@docusaurus/useBaseUrl';

<img src={useBaseUrl('img/integrations/saas-cloud/claude-compliance.png')} alt="Claude Compliance icon" width="50"/>

> The Claude Compliance Monitoring app for Sumo Logic provides centralized visibility into Claude platform activities, security events, user behavior, API usage, authentication activity, billing operations, integrations, data access, and organizational changes. The app enables security, compliance, and operations teams to monitor critical activities, detect suspicious behavior, investigate threats, and maintain governance across Claude environments. With dashboards covering Claude usage, API monitoring, authentication, exfiltration detection, integrations, SSO, session management, billing, user lifecycle events, and policy changes, the app delivers real-time operational insights and comprehensive compliance monitoring through detailed analytics and event correlation.

Reviewer: lead with Sumo Logic 

```suggestion
The Sumo Logic app for Claude Compliance provides security, compliance, and operations teams with centralized visibility into Claude platform activity, covering API usage, authentication events, billing operations, data access, integrations, SSO, and policy changes. Prebuilt dashboards and detection monitors help you identify suspicious behavior, investigate threats, and maintain governance across your Claude environment.
```

// Global filter
| where if("{{event_type}}" = "*",true,event_type matches "{{event_type}}")
| where if("{{organization}}" = "*",true,org_uuid matches "{{organization}}")
| where if("{{actor_type}}" = "*",true,actor_type matches "{{actor_type}}")

// Panel specific
| count by id,event_type
| count by event_type
| count
```

```sumo title="Messages by Role"
_sourceCategory="{{Logsdatasource}}"
| json "id", "updated_at", "user.id", "user.email_address", "name", "organization_id", "message.role", "message.content[0].type", "message.content[0].text", "message.model", "message.id" as id, updated_at, user_id, user_email, name, organization_id, role, message_type, message_value, model, message_id nodrop
| where user_email matches "{{user}}" or isBlank(user_email)
| where role matches "{{role}}" or isBlank(role)
| where model matches "{{model}}" or isBlank(model)
| where message_type matches "{{message_type}}" or isBlank(message_type)
| where message_value contains "{{message}}" or "{{message}}" == "*"
| where !isBlank(role)
| count by message_id, role
| count by role
| sort by _count, role
```

## Collection configuration

> This app uses the [Universal Connector](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/universal-connector-source) to collect **activity logs** from the Anthropic Compliance API and Sumo Logic's [Claude Compliance Source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/claude-compliance-source/) to collect chat messages logs from the Claude Compliance Messages API.

Reviewer: See [related comment](https://github.com/SumoLogic/sumologic-documentation/pull/6713/files#r3271123525)

Reviewer: stet - per @sumoanema
